src/lib/zkapp_command_builder/zkapp_command_builder.ml

```diff
@@ -10,27 +10,35 @@ let mk_forest ps :
 let mk_node account_update calls : _ Zkapp_command.Call_forest.Tree.t =
   { account_update; account_update_digest = (); calls = mk_forest calls }
 
-let mk_account_update_body authorization_kind call_type kp token_id
-    balance_change : Account_update.Body.Simple.t =
+let mk_account_update_body ?preconditions ?increment_nonce ?update
+    authorization_kind call_type kp token_id balance_change :
+    Account_update.Body.Simple.t =
   let open Signature_lib in
-  { update = Account_update.Update.noop
+  let preconditions =
+    Option.value preconditions
+      ~default:
+        Account_update.Preconditions.
+          { network = Zkapp_precondition.Protocol_state.accept
+          ; account = Account_update.Account_precondition.Accept
+          ; valid_while = Ignore
+          }
+  in
+  let increment_nonce = Option.value increment_nonce ~default:false in
```

Comment:
NIT: you could use the `let f ?(x = default)` syntax here to make this a bit cleaner.